### DISCUSSION TITLE
Shaders settings

### DISCUSSION THREAD

**Discussion by AlisterGontineac** [IrisShaders/Iris#455] [Q&A] [answered]
Firstly, FREAKING awesome! So glad I can use my beloved shaders on fabric now!

I'm using AstraLex (my Fav), but there is no option for editing the shader settings.

I tried moving the .txt file into the shaders folder, but Iris doesn't recognize the file. Is there a specific way to change the 
shader settings, or is this a function yet to come?

EDIT: Sorry, there is already a discussion about this

**Comment by WereWolfix** [comment, marked as answer]
I have found a way to change the settings from the shaders files. Obviously you must have unzipped the zip file. I give an example with BSL_shader. Go inside the BSL / shaders / lib folder, inside you will find a file called setting.glsl, open it with a text editor program. All the settings preceded by // will be disabled by default so if you want to have them active you have to delete the // same thing vice versa if you want to disable them add the // in front of the setting

**Reply by AlisterGontineac** [reply]
Been messing with this since you replied. Thank you!

**Reply by enagimintas** [reply]
Editing shader internal files breaks things. Some things are not supported by Iris yet. So you will not get support from official team with that. Please do not follow this method.

**Comment by enagimintas** [comment]
Settings are not supported yet. Coderbot and others are working on it.

**Reply by AlisterGontineac** [reply]
Alright, I'll wait patiently, thank you.
